build-logic/kotlin/src/main/kotlin/convention.kotlin-base.gradle.kts

```diff
@@ -4,15 +4,10 @@ plugins {
     id("convention.unit-testing")
 }
 
-dependencies {
-    add("implementation", "org.jetbrains.kotlin:kotlin-stdlib:1.5.20")
-}
-
 /**
- * 1.4 used because of kotlin dsl bundled version
- * todo use 1.5
+ * overrides 1.4, that comes from kotlin dsl bundled version
  */
-val kotlinLanguageVersion = "1.4"
+val kotlinLanguageVersion = "1.5"
```

Comment:
I'd good to make this change in a separate PR

Reply:
i agree, let me separate it